lib/chai/utils/addProperty.js

```diff
@@ -40,8 +39,9 @@ module.exports = function (ctx, name, getter) {
 
   Object.defineProperty(ctx, name,
     { get: function addProperty() {
+        var keep_ssfi = flag(this, 'keep_ssfi');
```

Comment:
imagine expect(true).to.be.false with false overwritten:

- expect sets ssfi
- to overwrites ssfi
- be overwrites ssfi
- custom false overwrites ssfi
- _super false overwrites ssfi

but actually that last part shouldn't happen because you don't want the _super in the stack trace. setting keep_ssfi during the _super prevents that from happening.